classes/script/visitor.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace mageekguy\atoum\script;
+
+use
+	mageekguy\atoum\script,
+	mageekguy\atoum\scripts
+;
+
+interface visitor
```

Comment:
the other visitor class , but in fact no? just same name but differnt purpose ;)